http/1.6/modules/ROOT/pages/http-about-http-connector-migration.adoc

```diff
@@ -2,25 +2,25 @@
 :keywords: http, https, request, requester
 :page-aliases: connectors::http/http-about-http-connector-migration.adoc
 
-Key changes in the HTTP connector for Mule Runtime 4.0 and later that can affect migration are:
-
-* The `http:listener-config` and `http:request-config` have a new, inner connection element.
-* The HTTP body option replaces the source parameter in the request and is incorporated to the Listener's response and error response.
-* The HTTP Listener and Request components return attributes, which include all previous properties.
-* HTTP parsing has been removed. DataWeave now handles special mime types. It can be used to both read and write such payloads.
-* The `static-resource-handler` is renamed `load-static-resource`.
-+
-* To access source HTTP Listener and HTTP Request attributes, the following DataWeave syntax replaces the Mule Expression Language (MEL) expressions for accessing past properties.
+The following list contains key changes for Anypoint Connector for HTTP (HTTP Connector) migration to Mule runtime engine (Mule) 4:
```

Comment:
Suggestion: The Mule 4 version of Anypoint Connector for HTTP (HTTP Connector) contains the following key differences from the Mule 3 version of the connector: 
(Not sure if this conveys what you are trying to say here.)